server/broadcast.go

```diff
@@ -97,7 +121,10 @@ func (pi *packetInfo) announceStart() {
 		player.isPassive = true
 
 		// Try to send the message to remaining players even if errors.
-		_ = writeMessage(player.conn, []*message.Signed{&m})
+		err := writeMessage(player.conn, []*message.Signed{&m})
+		if (err != nil) && debugMode {
+			fmt.Printf("[ANNOUNCE] Continuing to send after write error (%s) to (%s)\n", err, player)
```

Comment:
To be consistent.

`fmt.Printf("[Broadcast] Continuing to send after write error to %s: %s\n", getIP(player.conn), err)`